src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newObject = {};
+
+  for (const key in robot) {
+    if (newObject.hasOwnProperty(robot[key]) === true) {
```

Comment:
this explicit check redundant
```suggestion
    if (newObject.hasOwnProperty(robot[key])) {
```